examples/tutorials/advance_model_config/03_train_steps.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<p>Put names of fetched tensors in separate cell so that pipeline does not look that strange. You can use the top of the previous cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>